src/vmware/azext_vmware/_params.py

```diff
@@ -147,3 +148,29 @@ def load_arguments(self, _):
 
     with self.argument_context('vmware workload-network dhcp relay') as c:
         c.argument('server_addresses', nargs='+', validator=server_addresses_length, help='DHCP Relay Addresses. Max 3.')
+
+    with self.argument_context('vmware workload-network dns-service') as c:
+        c.argument('dns_service_id', help="NSX DNS Service identifier. Generally the same as the DNS Service's display name.")
+        c.argument('display_name', help='Display name of the DNS Service.')
+        c.argument('dns_service_ip', help='DNS service IP of the DNS Service.')
+        c.argument('default_dns_zone', help='Default DNS zone of the DNS Service.')

```

Comment:
I think it would be better to stick to lowercase `s` in `DNS service`.

Reply:
Updated for `DNS service` and `DNS zone` where this occurs.